.gitattributes

```diff
@@ -4,3 +4,6 @@
 /test/extensions/transport_sockets/tls/test_data/aes_128_key binary
 /test/extensions/transport_sockets/tls/test_data/ticket_key_* binary
 /test/**/*_corpus/* linguist-generated=true
+requirements.txt binary
```

Comment:
no we cant really - requirements.txt historically had more like the role of defining deps rather than pinning and can still be used for that but mostly people use as a lockfile and then requirements.in as the spec

i mean we possibly can, but now without making a stack of changes to a stack of things and hoping that everything is customizable

Reply:
also tbh it would be unexpected - `requirements.txt` is the lock file these days

Reply:
ok, thanks!